products/sle/main.pm

```diff
@@ -1316,6 +1316,13 @@ elsif (get_var('HPC')) {
         loadtest 'hpc/munge_slave'  if (check_var('HPC', 'munge_slave'));
     }
 }
+elsif (get_var('ROOTONLY')) {
+    load_boot_tests();
+    load_inst_tests();
+    load_reboot_tests();
+    loadtest "x11/xterm";
+    loadtest "x11/gnome_terminal";
```

Comment:
I see. I know that the SAP tests also run root-only so it might be worth to compare to that. I am wondering if it is not just as easy as defining the user as 'root'. Please compare to e.g. https://openqa.suse.de/tests/1023059# and https://openqa.suse.de/tests/1023062#